packages/node_modules/pouchdb-checkpointer/src/index.js

```diff
@@ -16,7 +16,7 @@ var LOWEST_SEQ = 0;
 function updateCheckpoint(db, id, checkpoint, session, returnValue) {
   return db.get(id).catch(function (err) {
     if (err.status === 404) {
-      if (db.type() === 'http') {
+      if (db.adapter === 'http' || db.adapter === 'https') {
```

Comment:
this is not really about remote vs not-remote; it's about warning the user when there's an http error in the console